tests/test_compressors/quantized_compressors/test_pack_quant.py

```diff
@@ -76,7 +76,7 @@ def test_quant_format(shape):
     dense_state_dict = {
         "dummy.weight": torch.rand(shape),
         "dummy.weight_scale": torch.tensor(0.01, dtype=torch.float32),
-        "dummy.weight_zero_point": torch.tensor(0, dtype=torch.int32),
+        "dummy.weight_zero_point": torch.tensor(0, dtype=torch.int8),
```

Comment:
aren't we pack-quantizing 4bit ints into 32 bit? why int8 here?

Reply:
Int4 isn't represented in torch. So we use int8s in an int4 range and then pack into int32